docs/decisions/0006-transmit-updates-only.rst

```diff
@@ -0,0 +1,87 @@
+`catalogs to transmit` under the customer's configurations (otherwise will default to all the customer's catalogs if
+`catalogs to transmit` is not provided) and request content metadata from the enterprise catalog. These catalogs can
+range from "all possible content" to a specific, single course.
+
+Decisions
+=========
+
+In order to address these performance issues and to more effectively handle on-boarding more customers to the Integrated
+Channels, we have decided to address requests to the enterprise catalog on two fronts:
+
+1. Increase the page size of requests going to `/api/v1/enterprise-catalog/<uuid>/get_content_metadata` from a base of
+10 to 100. This will reduce the number of calls necessary to retrieve the entire catalog from 150~ to about 15
+(as of 07/2021).
+
+2. Add an additional request/check to the Integrated Channel's exporter that will retrieve the last time either a
+catalog or the associated content metadata of a catalog has been changed (whichever is most recent). This timestamp will
+be compared to the most recent time the customer has had a successful transmission of that particular catalog. If there
+is no updated needed, then the exporter will refrain from querying enterprise catalog for the catalog's metadata.
+
+Exceptions
+==========
+
+All CSOD customers have been told to expect the cornerstone/course-list endpoint will always return a current state of
+the course catalog, meaning that any missing content will be treated as deletes. It is also important to note that
+before this ADR, we were not saving any audits of CSOD related transmissions. As such, there are CSOD specific
+exemptions to this ADR- specifically we are:
+
+1. adding an export_force_all_catalogs method to the content metadata exporter that effectively functions exactly the
+same as the current export method, ignoring updates needed and transmiting all catalogs.
+
+2. adding a new view endpoint `course-updates` that functions exactly as the existing `course-list` except it will
+exclude metadata from catalogs that do not require updates.
+
+3. maintaining the existing export behavior in the `course-list` endpoint
+
+4. making create, update and deletes of ContentMetadataItemTransmission objects on all CSOD transmissions the same way
+all other channels currently do.
+
+Consequences
+============
+
+The transmit-content-metadata jenkins job can continue to run every 10 minutes, so that it catches and transmits updated
+content metadata when the metadata actually changes. However, the cost of the average run in terms of CPU usage will go
+down significantly. That being said, f any metadata associated with a catalog has changed between job run N and N+1,
+then we’ll transmit the full catalog to the integrated channel during run N+1. Typically, the metadata in
+enterprise-catalog is updated once per day, so we’d expect to only be transmitting catalog data to these integrated
```

Comment:
Lines 82-87 are business processes, they should not go in the ADR. If we change the exception for Cornerstone in the future, we can write a new ADR.

Reply:
I've rewritten the `Consequences` section as I feel it was more Alex D's and my separate thoughts copied into a block. It should now read more cleanly.